docs/user/using-the-capture-app.md

```diff
@@ -789,6 +789,16 @@ You will find the predefined list views above the filters for the list. Click to
 
 ![](resources/images/tei_list_predefined_views.png)
 
+## Tracker program stage working list
+
+Visualize data elements from one single stage in the same working list. To do this, you choose “Program stage” under “More filters" and then a program stage option. 
```

Comment:
I think we might want to avoid using the word "visualize", as we have so many apps that handle data visualization.
Suggestion:
`You can show data elements from a single stage in a working list. Select the "Program stage" option from the "More filters" dropdown, then choose a program stage.`